docs/contributors/coding-guidelines.md

```diff
@@ -90,3 +90,63 @@ We want our font sizes to be declared with rem or em units instead of hardcoded
 We have a mixin named `font-size()` that given a number of the font size in px, it converts it to rem. It accepts a second parameter which is the line height, it will be divided by the font-size and the result will be the relative units for the `line-height` CSS property.
 
 In parallel to that, consider whether other size/distance units in your CSS need to be rem/em instead of px. In general, rem/em should be preferred if it doesn't break the design with big font sizes. There is another mixin named `rem()` that helps converting px units to rem (given a px size and optionally a base size).
+
+### CSS specificity wars with 3rd party popular themes
+
+We want our blocks to look good with as many themes as possible out of the box. Some times, our styles will conflict with theme styles that have higher specificity. In those cases, we might be tempted to increase the specificity of our selectors, but increasing it too much would make it harder for other themes to style our blocks. In the following lines, there is a set of rules that might help deciding when to increase specificity or not. They are not hard rules so feel free to apply your best judgement every time.
```

Comment:
```suggestion
We want our blocks to look good with as many themes as possible out of the box. Sometimes our styles will conflict with theme styles that have higher specificity. In these cases it may be tempting to increase the specificity of selectors, but increasing them too much makes it harder for other themes to style our blocks. 

The following guidelines should help you decide _when_ to increase specificity, if at all. They are not hard rules so feel free to apply your best judgement on a case-by-case basis.
```